lib/canary/plugs.ex

```diff
@@ -414,26 +413,31 @@ defmodule Canary.Plugs do
     end
   end
 
-  defp handle_unauthorized(conn = %{skip_canary_handler: true}, _opts),
+  defp handle_unauthorized(%{skip_canary_handler: true} = conn, _opts),
```

Comment:
I like the yoda syntax, but if we are going to use it here, it should be used throught the whole code base for consistency, e.g. https://github.com/cpjk/canary/pull/54/files#diff-3938ea297feb0d441ffe05cec9da28eaR282